src/Samples/NewWindow/Program.fs

```diff
@@ -112,12 +112,21 @@ module App =
   ]
 
 
-[<EntryPoint; STAThread>]
-let main _ =
-  let createWindow2 () = Window2(Owner = Application.Current.MainWindow)
-  let bindings = App.mainBindings Window1 createWindow2
+let fail _ = failwith "never called"
+let mainVm = ViewModel.designInstance (App.init ()) (App.mainBindings fail fail ())
+let window1Vm = ViewModel.designInstance (App.init ()) (App.window1Bindings ())
+let window2Vm = ViewModel.designInstance App.initWindow2 (App.window2Bindings ())
+
+
+let main (mainWindow, (createWindow1: Func<#Window>), (createWindow2: Func<#Window>)) =
```

Comment:
AFAIK you don't have to pass a tuple here; If you pass them as normal curried F# args, then the call from C# will still be the same.

Reply:
Yes, you are correct.  Thanks for the reminder.